containerz/containerz.proto

```diff
@@ -299,3 +327,69 @@ message LogRequest {
 message LogResponse {
   string msg = 1;
 }
+
+message CreateVolumeRequest {
+  // name of the volume. If present, this system must create the volume with
+  // this name otherwise it must allocate a name.
+  string name = 1;
+
+  // The volume driver. If not specified, this defaults to the the standard
+  // linux mount operation (http://man7.org/linux/man-pages/man8/mount.8.html)
+  // and associated options.
+  string driver = 2;
+
+  // Options for the driver. The actual option keys and values are driver
+  // specific.
+  map<string, string> options = 3;
+
+  // Labels to apply to the volume. Labels are metadata for the the volume.
```

Comment:
Can you explain what the use case for this is?

Reply:
This allows to label/tag volumes with information about what they contain or even  #perhaps version information (e.g. tagging the config version allowing rollbacks to another version more easily) which can then be filtered.